examples/bricks/CardPayment/0-Simple.tsx

```diff
@@ -1,13 +1,11 @@
 import React from 'react';
-import Card from '../../../src/bricks/cardPayment';
-
-import initMercadoPago from '../../../src/mercadoPago/initMercadoPago';
+import { initMercadoPago, CardPayment } from '../../../src/index';
```

Comment:
is it still possible to import the functions/components the old way? otherwise, i think that it should be a major bump 🤔